src/Utilities/tests/test_project_data.cpp

```diff
@@ -38,23 +38,19 @@ TEST_CASE("ProjectData", "[ohmmsapp]")
   REQUIRE(proj1.getTitle().size() > 0);
   REQUIRE(proj1.getTitle() == proj1.getName());
 
-
-  ProjectData proj2("test");
+  ProjectData proj2;
```

Comment:
We need to keep pass title via constructor.

Reply:
As noted this never worked. See below. If you passed the name you got that as the class name (never used) and title was always "asample"

Reply:
The old code was just bad. I meant we need a way to set the title without relying on XML.
I can image in a driver unit test, we may check certain composite names (used for output). Having a convenient way will be helpful. The intention of existing setTitle was unclear and it was never called. So I just removed it.